### PR TITLE
added the addParentsPom to the maven command

### PR DESCRIPTION
**Pull request by joratp**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
This is the change we are adding to fix the mvn command we run to copy the dependencies, currently this command fails for some applicatioins where they have first party dependencies + multi-module. This should help with that case. Has been tested on some multi module cases and works. 

## Checklist
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by joratp** [comment]
Based on some of our error cases we are getting number of issues where we get the following error:

"The requested module could not be built in Java 8 sandbox. Please ensure that code builds in your local environment before retrying. In some cases, the transformation cannot be supported."

we think that this is due to missing parentPom or references to parent Pom that we did not capture in our command using the command provided currently in the code.

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt [hunk 1]
```diff
@@ -133,7 +133,8 @@ data class CodeModernizerSessionContext(
                 "dependency:copy-dependencies",
                 "-DoutputDirectory=$destinationDir",
                 "-Dmdep.useRepositoryLayout=true",
-                "-Dmdep.copyPom=true"
+                "-Dmdep.copyPom=true",
+                "-Dmdep.addParentPoms=true"
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt:137]: test?

**Comment by dhasani23** [comment]
This change is needed because the pom.xml build file of the parent must be included in the zipped code (children can inherit dependencies from parent in multi-module Maven projects) so that our backend is able to get all necessary dependencies and build the project successfully. This new line which adds a flag to the mvn command was needed from the beginning but was accidentally left out; it has always been included in the corresponding VS Code implementation.

**Review by dhasani23 — APPROVED** [review]

File: .changes/next-release/bugfix-e7224ad0-f4a7-4ef5-aba8-6130c235dab2.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "This change is needed because the pom.xml build file of the parent must be included in the zipped code (children can inherit dependencies from parent in multi-module Maven projects) so that our backend is able to get all necessary dependencies and build the project successfully. This new line which adds a flag to the mvn command was needed from the beginning but was accidentally left out; it has always been included in the corresponding VS Code implementation."
```
Comment (manodnyab) [inline .changes/next-release/bugfix-e7224ad0-f4a7-4ef5-aba8-6130c235dab2.json:3]: Can the description be more concise eg format: Fix to ensure backend gets necessary dependencies

Comment (joratp) [reply .changes/next-release/bugfix-e7224ad0-f4a7-4ef5-aba8-6130c235dab2.json]: Updated in the new commit

**Review by manodnyab — APPROVED** [review]

**Pull request merged**